docs/context/ref/SearchCriteria.md

```diff
@@ -0,0 +1,60 @@
+---
+id: SearchCriteria
+sidebar_label: SearchCriteria
+title: SearchCriteria
+hide_title: true
+---
+# `SearchCriteria`
+
+A context type representing a search criteria that can be used to filter and display messages from a chat.
+
+## Type
+
+`fdc3.searchCriteria`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/searchCriteria.schema.json
+
+## Details
+
+| Property         | Type            | Required | Example Value        |
+|------------------|-----------------|----------|----------------------|
+| `type`           | string          | Yes      | `'fdc3.searchCriteria'`     |
+| `contexts`    | (Instrument&#124;<br>Contact&#124;<br>Organization&#124;<br>string)[]  | Yes      | <pre>[<br>&emsp;&emsp;{<br>&emsp;&emsp;&emsp;&emsp;"type": "fdc3.instrument",<br>&emsp;&emsp;&emsp;&emsp;"id": {<br>&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;"ticker": "AAPL"<br>&emsp;&emsp;&emsp;&emsp;}<br>&emsp;&emsp;},<br>&emsp;&emsp;{<br>&emsp;&emsp;&emsp;&emsp;"type": "fdc3.contact",<br>&emsp;&emsp;&emsp;&emsp;"name":"Jane Doe",<br>&emsp;&emsp;&emsp;&emsp;"id": {<br>&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;"email": "jane.doe@mail.com"<br>&emsp;&emsp;&emsp;&emsp;}<br>&emsp;&emsp;},<br>&emsp;&emsp;{<br>&emsp;&emsp;&emsp;&emsp;"type": "fdc3.organization",<br>&emsp;&emsp;&emsp;&emsp;"name":"Symphony",<br>&emsp;&emsp;},<br>&emsp;&emsp;"#OrderID45788422",<br>]</pre> |
```

Comment:
Does this example need to be revisited? removing the html entities.

Reply:
Hi @mistryvinay 
I removed the HTML entities from the example.

Reply:
Actually, the HTML entities are there to make it render on the web. Markdown has some shortcomings when it comes to code examples...

Heres what it looks like in github's preview without the HTML entities:
![image](https://user-images.githubusercontent.com/1701764/187876345-a814bddf-73da-4a03-8a9f-8a96684d0d44.png)

and with them:
![image](https://user-images.githubusercontent.com/1701764/187876615-35dbfea0-e229-4c74-97ad-c923e1297b6f.png)

Use the rich preview (the file icon) to see the markdown rendered: 
![image](https://user-images.githubusercontent.com/1701764/187876766-1e9ce4f3-e898-479a-8305-9e1c51b18d8a.png)

We need to have another go at asking FINOS to adjust the netlify config to ensure it builds previews for PRs like this - right now it only builds if it detects changes in the /website directory...

Reply:
OK. @kriswest Do you want me to restore the changes (HTML entities)?

Reply:
Managed to track down the preview - still showing state before they were removed: 
![image](https://user-images.githubusercontent.com/1701764/187898932-bd3e9a9b-186b-4f72-b773-4f0d63e87c86.png)

If you merge master into your PR (needs doing anyway) it should refresh so you can see it without at: https://deploy-preview-797--lambent-kulfi-cf51a7.netlify.app/docs/next/context/ref/searchcriteria (once the deploy has run).

From memory docusaurus really struggles with long code lines in tables. Might be worth a check first in case I'm mis-remembering.